src/utils/date-time-formatter.js

```diff
@@ -385,18 +385,22 @@ DateTimeFormatter.getTokenDefinitions = function () {
             index: 0,
             extract () { return '(\\d{2})'; },
             parser (val) {
+                let result;
                 if (val) {
                     const l = val.length;
                     val = val.substring(l - 2, l);
                 }
                 let parsedVal = DateTimeFormatter.defaultNumberParser()(val);
                 let presentYear = Math.trunc(((new Date()).getFullYear()) / 100);
                 if (parsedVal instanceof Number) {
-                    parsedVal = (presentYear * 100) + parsedVal;
```

Comment:
Why `instanceof` Number check? As I can see, `defaultNumberParser` always returning number?

Reply:
`defaultNumberParser` return original value if the value sent to be parsed cannot be parsed.
So the check.